packages/connectors/ledger-connector/src/iframe/helpers.ts

```diff
@@ -1,26 +1,15 @@
-export const isIframe = (): boolean => {
+export const isIframe = () =>
+  globalThis.window && globalThis.window.parent !== globalThis.window;
+
+export const isLedgerLive = () => {
+  if (typeof window === 'undefined') return false;
+
   try {
```

Comment:
Previously, there was a variable (state) that was a kind of cache.

Reply:
This is an ultra cheap calculation, no reason to be cached.